src/LocalFileServer/index.ts

```diff
@@ -65,7 +65,12 @@ export class LocalFileServer {
 
     this.router
       .get(routePattern, async ({ response, request, logger }) => {
-        const location = request.param(LocalFileServer.filePathParamName).join('/')
+        let location = request.param(LocalFileServer.filePathParamName).join('/')
+        try {
+          location = decodeURIComponent(location)
```

Comment:
Why this line would throw an error?

Reply:
while trying to access the URL be malformed, it will throw an error 
```
Uncaught URIError: URI malformed
```

So I think it should throw an error

Reply:
Yup, that exception should be raised as it is